CHANGELOG.md

```diff
@@ -1,18 +1,18 @@
+# 2.13.0 

```

Comment:
2.12.0 was never released, so add these release notes after the current 2.12.0 release notes.